src/core/core.scale.js

```diff
@@ -344,9 +355,12 @@ var Scale = Element.extend({
 		me.beforeFit();
 		me.fit();
 		me.afterFit();
-		//
+		// Auto-skip
+		me._ticksToDraw = tickOpts.display && tickOpts.autoSkip ? me._autoSkip(me.getTicks()) : ticks;
```

Comment:
Why don't we use the same ticks source (`me.getTicks()` vs `ticks`)? At least, I would use `me._ticks` instead of `me.getTicks()`. But actually, there is no guarantee that `this._ticks` never change between this line and when it's assigned at line `342`, in which case `ticks` would be outdated. It may be safer to always use `me._ticks`:

```javascript
me._ticksToDraw = tickOpts.display && tickOpts.autoSkip ? me._autoSkip(me._ticks) : me._ticks;
```

Reply:
Done